controls/radvirtualgrid/mvvm.md

```diff
@@ -0,0 +1,327 @@
+	            {
+	                int propertyIndex = this.GetPropertyIndex("Name");
+	
+	                if (e.Index == propertyIndex)
+	                {
+	                    e.Value = "Club Name";
+	                }
+	                else
+	                {
+	                    base.OnHeaderValueNeeded(e);
+	                }
+	            }
+	        }
+	
+	        protected override void OnEditorNeeded(EditorNeededEventArgs args)
+	        {
+	            int propertyIndex = this.GetPropertyIndex("Established");
+	            if (args.ColumnIndex == propertyIndex)
+	            {
+	                var editor = new RadDateTimePicker();
+	                var item = (this.Source.GetItemAt(args.RowIndex) as Club);
+	
+	                args.Editor = editor;
+	                editor.SelectedValue = item.Established;
+	                args.EditorProperty = RadDateTimePicker.SelectedValueProperty;
+	            }
+	            else
+	            {
+	                base.OnEditorNeeded(args);
+	            }
+	            
+	        }
+	
+	        protected override bool IsColumnReadOnly(int columnIndex)
+	        {
+	            int propertyIndex = this.GetPropertyIndex("Name");
+	            if (columnIndex == propertyIndex)
+	            {
+	                return true;
+	            }
+	
+	            return base.IsColumnReadOnly(columnIndex);
+	        }
+	
+	        /// <summary>
+	        /// Gets the index of a given property
+	        /// </summary>
+	        /// <param name="propertyName">The property name</param>
+	        /// <returns>The property index</returns>
+	        private int GetPropertyIndex(string propertyName)
+	        {
+	            ItemPropertyInfo itemProperty = this.ItemProperties.Where(ip => ip.Name == propertyName).FirstOrDefault();
+	            int index = this.ItemProperties.IndexOf(itemProperty);
+	
+	            return index;
+	        }
+	
+	        /// <summary>
+	        /// Persists the selected indexes
+	        /// </summary>
+	        /// <param name="selectedIndexes"></param>
+	        private void PersistSelection(IEnumerable<int> selectedIndexes)
+	        {
+	            foreach (int index in selectedIndexes)
+	            {
+	                this.selectedIndexes.Add(index);
+	            }
+	        }
+	
+	        /// <summary>
+	        /// Adds the previously persisted indexes
+	        /// </summary>
+	        /// <param name="selectedIndexes"></param>
+	        private void UpdateSelection(IEnumerable<int> selectedIndexes)
+	        {
+	            foreach (int index in selectedIndexes)
+	            {
+	                if (!this.ParentGrid.SelectedIndexes.Contains(index))
+	                {
+	                    this.ParentGrid.ToggleIndexSelection(index);
+	                }
+	            }
+	        }
+    }
+{{endregion}}
+
+## Defining the View Model
+
+Furthermore, the custom DataProvider needs to be exposed by a view model.
```

Comment:
Same comment as for the last section. Start with something more generic instead of "Furthermore, ". For example "This section shows how to define a view model that holds the custom DataProvider."